### PR TITLE
Improvement for character escape handling

### PR DESCRIPTION
**Pull request by ssharma**
https://issues.apache.org/jira/browse/CAMEL-10219

### CODE REVIEW

File: components/camel-jaxb/src/main/java/org/apache/camel/converter/jaxb/JaxbDataFormat.java [hunk 1]
```diff
@@ -64,6 +64,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.sun.xml.bind.marshaller.CharacterEscapeHandler;
```
Comment (davsclaus) [inline components/camel-jaxb/src/main/java/org/apache/camel/converter/jaxb/JaxbDataFormat.java]: We cannot have dependency on com.sun classes.

File: components/camel-jaxb/src/main/java/org/apache/camel/converter/jaxb/JaxbConstants.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.converter.jaxb;
+
+/**
+ * Constants used by Camel Jaxb module
+ * 
+ * @author Siddharth Sharma
```
Comment (davsclaus) [inline components/camel-jaxb/src/main/java/org/apache/camel/converter/jaxb/JaxbConstants.java]: Please remove author tag from source code

File: components/camel-jaxb/src/main/java/org/apache/camel/converter/jaxb/JaxbDataFormat.java (hunk 1 above)
Comment (ssharma) [inline components/camel-jaxb/src/main/java/org/apache/camel/converter/jaxb/JaxbDataFormat.java]: Sure. Sorry, wasn't aware of the dependency constraint. I've updated the PR and made it generic to inject any properties for a given JAXB-RI

File: components/camel-jaxb/src/main/java/org/apache/camel/converter/jaxb/JaxbConstants.java (hunk 2 above)
Comment (ssharma) [inline components/camel-jaxb/src/main/java/org/apache/camel/converter/jaxb/JaxbConstants.java]: Done! Learning about OSS best practices. Thanks!

**Comment by ssharma** [comment]
@davsclaus  Updated PR and incorporated review comments

**Pull request closed without merging**